packages/itwinui-css/src/tile/tile.scss

```diff
@@ -87,6 +87,35 @@
       -webkit-line-clamp: 2;
     }
   }
+
+  //#region Statuses
+  &.iui-uploading {
```

Comment:
Maybe use more generic name like `iui-progress`

Reply:
Fixed

Reply:
Forgot to push? 😀